### PR TITLE
Feature/redshiftlogging

### PR DESCRIPTION
**Pull request by vigneswararaomacharla**
Feature/redshiftlogging.

### CODE REVIEW

File: deployment/cf-templates/identification-nested.json [hunk 1]
```diff
@@ -0,0 +1,267 @@
+{
```
Comment (oleksii-dowjones) [inline deployment/cf-templates/identification-nested.json]: Unrelated changes. Can we somehow get rid of them?

File: deployment/cf-templates/reporting-remediation-crossaccount-role.json [hunk 2]
```diff
@@ -138,6 +138,23 @@
...
+                                    "Sid": "RedshiftIssues",
+                                    "Effect": "Allow",
+                                    "Action": [
+                                        "redshift:DescribeClusterSecurityGroups",
+                                        "redshift:DescribeClusterParameterGroups",
+                                        "redshift:DescribeLoggingStatus",
+                                        "redshift:GetClusterCredentials",
+                                        "redshift:DescribeClusters",
+                                        "redshift:EnableLogging",
+                                        "redshift:DisableLogging",
```
Comment (oleksii-dowjones) [inline deployment/cf-templates/reporting-remediation-crossaccount-role.json]: Why do we need to disable logging?

File: deployment/cf-templates/reporting-remediation-role.json [hunk 3]
```diff
@@ -206,6 +206,23 @@
...
+                                    "Sid": "RedshiftIssues",
+                                    "Effect": "Allow",
+                                    "Action": [
+                                        "redshift:DescribeClusterSecurityGroups",
+                                        "redshift:DescribeClusterParameterGroups",
+                                        "redshift:DescribeLoggingStatus",
+                                        "redshift:GetClusterCredentials",
+                                        "redshift:DescribeClusters",
+                                        "redshift:EnableLogging",
+                                        "redshift:DisableLogging",
```
Comment (oleksii-dowjones) [inline deployment/cf-templates/reporting-remediation-role.json]: Again, why do we need to disable logging?

File: hammer/reporting-remediation/reporting/create_redshift_logging_issue_tickets.py [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+                            msg=f"{comment}"
+                                f"{' (' + jira.ticket_url(issue.jira_details.ticket) + ')' if issue.jira_details.ticket else ''}",
+                            owner=owner,
+                            account_id=account_id,
+                            bu=bu, product=product,
+                        )
+                        IssueOperations.set_status_closed(ddb_table, issue)
+                    # issue.status != IssueStatus.Closed (should be IssueStatus.Open)
+                    elif issue.timestamps.updated > issue.timestamps.reported:
+                        logging.error(f"TODO: update jira ticket with new data: {table_name}, {account_id}, {cluster_id}")
```
Comment (oleksii-dowjones) [inline hammer/reporting-remediation/reporting/create_redshift_logging_issue_tickets.py]: Why do we need it?

File: deployment/cf-templates/reporting-remediation-crossaccount-role.json (hunk 2 above)
Comment (vigneswararaomacharla) [reply deployment/cf-templates/reporting-remediation-crossaccount-role.json]: Removed unnecessary permissions.

File: deployment/cf-templates/reporting-remediation-role.json (hunk 3 above)
Comment (vigneswararaomacharla) [reply deployment/cf-templates/reporting-remediation-role.json]: Removed unnecessary permissions.
